jinishakheni/BrainBash-Frontend
# Implement filter on event list page

What has been implemented:
- Implement filter functionality on event list page

How it works:
- Display all events, if filter is not selected.
- Display events based on filter selected.
- Search event based on filter selection.

Test:
- On page load, it should display all events.
- Search without filter selection
- Search with filter selection

closes #26
